PaddleCV/object_detection/configs/self_faster-rcnn_ResNet50-ESFPN.yml

```diff
@@ -0,0 +1,255 @@
+architecture: FasterRCNN
+train_feed: FasterRCNNTrainFeed
+test_feed: FasterRCNNTestFeed
+eval_feed: FasterRCNNEvalFeed
+fusebn: false
+pretrain_weights: /home/vis/gry/to_ygh/pretrain/
+# weights: /home/vis/gry/train/trained_models/coco/S02/model_final/
+weights: /home/vis/gry/to_ygh/faster_rcnn_r50_esfpn_2x/
+# /home/vis/gry/train/trained_models/coco/S158/model_final/
```

Comment:
remove the un-used code, change weights dir: output/faster_rcnn_r50_esfpn_1x/model_final